spec/factories/restaurant.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :restaurant do
+    name{Faker::Name.name}
+    cuisine {Faker::Name.name}
+    rating {0}
+    address {Faker::Address}
+    max_delivery_time {Faker::Number.rand}
```

Comment:
What range does this random number give? What range do you want max_delivery_time to be? Is max_delivery_time discrete or continuous?